sass/custom/_terminology_tooltip.scss

```
@@ -0,0 +1,46 @@
+.terminology {
+  position: relative;
+  border-bottom: 2px dotted #049cdb;
```

```suggestion
  border-bottom: 2px dotted $primary-color;
```
We should use the variable instead of hard-coding the value.
<https://github.com/home-assistant/home-assistant.io/blob/current/sass/oscailte/_variables.scss#L53>